packages/xsnap/src/snapStore.js

```diff
@@ -130,5 +130,13 @@ export function makeSnapStore(
     }, `${hash}-load`);
   }
 
-  return freeze({ load, save });
+  /**
+   * @param {string} hash
+   * @throws { Error } if there is no such snapshot
+   */
+  function deleteSnapshot(hash) {
```

Comment:
it might be nice to assert there are no `/` in the alleged `hash`, to limit the authority of this function to deleting things under `root`